tests/base/array-spread.js

```diff
@@ -0,0 +1,7 @@
+const fruits = ['apple', 'banana'];
+
+const eadibles = [...fruits];
+
+const envoke = () => eadibles;
```

Comment:
did you mean invoke? I'm not 100% sure that envoke is a real english word, but I might be wrong 😸